website/blog/2022-11-08-pjson-how-to-store-bson-in-jsonb.md

```diff
@@ -32,8 +32,8 @@ Here the hexadecimal `\x00` notation represents `0000 0000` in bits and similarl
 In BSON, `\x00` is a byte used as a terminator to indicate the end of a document.
 In BSON, field names use `cstring` which are UTF-8 characters followed by `\x00`.
 
-```json
-{ "foo": "bar" }
+```json5
+{ foo: 'bar' }
```

Comment:
I think we should not make unrelated changes in this PR. Both forms should work.